### PR TITLE
NIP44 Encryption Support

### PR DESCRIPTION
**Pull request by pavanjoshi914**
1. NIP-44 encryption and decryption so that users can access nostr features built on the new primitives (derived from https://github.com/getAlby/lightning-browser-extension/pull/2653)
2. extract NIP44 into the extension itself until we update js-sdk to nostr 2.x

### CODE REVIEW

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -1176,18 +1176,21 @@
...
-    },
     "bitcoin": {
       "getaddress": "Read your Bitcoin receive address"
     },
     "liquid": {
       "getaddress": "Read your Liquid receive address",
       "signschnorr": "Sign message with your key"
     },
+    "nostr": {
+      "getpublickey": "Read your public key.",
```
Comment (reneaaron) [inline src/i18n/locales/en/translation.json:1187]: Remove those dots pls

**Review by reneaaron — APPROVED** [review]
Please fix the translations, otherwise: 

utACK

File: src/i18n/locales/en/translation.json (hunk 1 above)
Comment (pavanjoshi914) [reply src/i18n/locales/en/translation.json]: resolved

File: src/extension/background-script/nostr/index.ts [hunk 2]
```diff
@@ -69,6 +82,14 @@ class Nostr {
     return Utf8.stringify(decrypted);
   }
 
+  nip44Encrypt(peer: string, plaintext: string) {
```
Comment (rolznz) [inline src/extension/background-script/nostr/index.ts:85]: Can the existing encrypt and functions be prefixed with `nip04`?

**Review by rolznz — APPROVED** [review]
utACK

File: src/extension/background-script/nostr/index.ts (hunk 2 above)
Comment (pavanjoshi914) [reply src/extension/background-script/nostr/index.ts:85]: resolved

**Comment by alexgleason** [comment]
Hey y'all can you do a release please? Wanting to nip44 encrypt events but it only works if people can decrypt them. Thank you! :orange_heart:

**Comment by pavanjoshi914** [comment]
Yeah its already present in the production!

**Pull request merged**